src/ProjectTemplates/test/GrpcTemplateTest.cs

```diff
@@ -24,11 +25,16 @@ public GrpcTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelper
         public ProjectFactoryFixture ProjectFactory { get; }
         public ITestOutputHelper Output { get; }
 
-        [ConditionalFact(Skip = "This test run for over an hour")]
+        [ConditionalFact]
         [SkipOnHelix("Not supported queues", Queues = "Windows.7.Amd64;Windows.7.Amd64.Open;OSX.1014.Amd64;OSX.1014.Amd64.Open")]
         [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/19716")]
         public async Task GrpcTemplate()
         {
+            // Setup AssemblyTestLog
+            var assemblyLog = AssemblyTestLog.Create(Assembly.GetExecutingAssembly(), baseDirectory: Project.ArtifactsLogDir);
```

Comment:
Why are we doing this here instead of the normal way?

Reply:
Glad you asked. Normally, we would be using `LoggedTest` but that requires the use of [`AspNetTestFramework`](https://github.com/dotnet/aspnetcore/blob/master/src/Testing/src/build/Microsoft.AspNetCore.Testing.props#L18-L21) xunit test framework. This cannot be used in template tests since they already use their own test framework: https://github.com/dotnet/aspnetcore/blob/9f1a978230cdd161998815c425bfd2d25e8436b6/src/ProjectTemplates/test/AssemblyInfo.AssemblyFixtures.cs#L8 and so we have to do it the oldschool way.

I think it's possible to combine the logic of the two xunit frameworks but that's a rather large undertaking that doesn't seem appropriate here.